components/proxy/src/main/java/com/hotels/styx/proxy/plugin/InstrumentedPlugin.java

```diff
@@ -42,21 +43,23 @@
     private static final Logger LOGGER = getLogger(InstrumentedPlugin.class);
 
     private final NamedPlugin plugin;
-    private final SimpleCache<HttpResponseStatus, Meter> errorStatusMetrics;
-    private final SimpleCache<Class<? extends Throwable>, Meter> exceptionMetrics;
-    private final Meter errors;
+    private final SimpleCache<HttpResponseStatus, Counter> errorStatusMetrics;
+    private final SimpleCache<Class<? extends Throwable>, Counter> exceptionMetrics;
+    private final Counter errors;
 
     public InstrumentedPlugin(NamedPlugin plugin, Environment environment) {
         this.plugin = requireNonNull(plugin);
         requireNonNull(environment);
 
+        Metrics.addRegistry(environment.meterRegistry());
+
         this.errorStatusMetrics = new SimpleCache<>(statusCode ->
-                environment.metricRegistry().meter("plugins." + plugin.name() + ".response.status." + statusCode.code()));
+                Metrics.counter("plugins." + plugin.name() + ".response.status." + statusCode.code()));
```

Comment:
I'd suggest using a tag for the plugin name, rather than prefixing the name with "plugins.<name>". 
Perhaps also the status code should be a tag, but I'm less sure about this.
Then you end up with the meter just being a counter called "plugin.response" with tags for plugin name and status code. WDYT?

Reply:
To be idiomatic with the usage of tags, I think this change would be reasonable.